src/validators.ts

```diff
@@ -0,0 +1,99 @@
+const maxSignedInt32 = 2147483647
+
+const isInt = (value:number) => typeof value == 'number' && Number.isInteger(value)
```

Comment:
Is `value:number` correct? In theory I could be passing `"goose"` to an `isInt()` call, which would obviously fail.

Should all of these functions have `any` instead to reflect what they _may_ be passed, rather than what they'd like in a "passing" case?

Reply:
No bloody idea.